src/index.rs

```diff
@@ -1,3 +1,4 @@
+use std::collections::BTreeSet;
```

Comment:
move this use statement into the other ones

Reply:
good catch